packages/schemas/src/foundations/jsonb-types.ts

```diff
@@ -186,12 +186,12 @@ export const identitiesGuard = z.record(identityGuard);
 export type Identity = z.infer<typeof identityGuard>;
 export type Identities = z.infer<typeof identitiesGuard>;
 
-const baseVerification = {
+export const baseVerification = {
   id: z.string(),
   createdAt: z.date(),
 };
 
-const verificationGuard = z.discriminatedUnion('type', [
+export const verificationGuard = z.discriminatedUnion('type', [
```

Comment:
Better give a more specific name.  

`import {verificationGuard, baseVerification} from @logto/schemas` 

Hard to tell what verification stands for without any context.

Reply:
updated